integration/shareddata/composites.go

```diff
@@ -414,6 +414,10 @@ var ArrayDocuments = &Values[string]{
 			}},
 		},
 		"array-three-documents": bson.A{
+			bson.D{{
+				"zoo",
+				bson.A{bson.D{{"a", "b"}}},
+			}},
```

Comment:
Using `{$sort: -1}` on `ArrayDocuments` collection, `array-three-documents` and `array-two-documents` results in the same comparison order. Because array comparison uses the largest element from the array and both contain the same largest array `bson.D{{"foo", bson.A{bson.D{{"bar", "hello"}}}}}`. This cause not deterministic sorting in `$group` with group key `$v` which aggregates `_id` as the value, and then uses the comparison order to sort response. So compat and target sorted in wrong order.

Adding a larger value here forces comparison order of `array-three-documents` to be greater than `array-two-documents`